cmd/kubeadm/app/cmd/reset.go

```diff
@@ -183,24 +177,31 @@ func resetWithDocker(execer utilsexec.Interface, dockerCheck preflight.Checker)
 func resetWithCrictl(execer utilsexec.Interface, dockerCheck preflight.Checker, criSocketPath, crictlPath string) {
 	if criSocketPath != "" {
 		fmt.Printf("[reset] Cleaning up running containers using crictl with socket %s\n", criSocketPath)
-		listcmd := fmt.Sprintf(crictlSandboxesParamsFormat, crictlPath, criSocketPath)
-		output, err := execer.Command(listcmd).CombinedOutput()
+		fmt.Println("[reset] listing running pods using crictl")
+		params := []string{"-r", criSocketPath, "pods", "--quiet"}
+		fmt.Printf("[reset] executing command %s %s\n", crictlPath, strings.Join(params, " "))
```

Comment:
[2]
so ideally the whole command should be in quotes using `%q`, but this is OK too.
change if see fit.